src/Microsoft.VisualBasic.Core/tests/Microsoft/VisualBasic/Devices/ComputerTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.VisualBasic.Devices.Tests
+{
+    public class ComputerTests
+    {
+        [Fact]
+        public void Properties()
+        {
+            var computer = new Computer();
+
+            var audio = computer.Audio;
+            Assert.NotNull(audio);
+            Assert.Same(audio, computer.Audio);
```

Comment:
I'm not sure what these tests are actually proving?

Reply:
The tests are simply verifying the instances returned are non-null and the same instance each time. The properties are very simple so there's probably not much else to test.

---
In reply to: [277352257](https://github.com/dotnet/corefx/pull/37085#discussion_r277352257) [](ancestors = 277352257)